src/cogs/esports/slash/scrims.py

```diff
@@ -36,7 +36,7 @@ async def can_use_command(self, interaction: discord.Interaction) -> bool:
         return True
 
     @app_commands.command()
-    @app_commands.describe(user="The user you want to ban from scrims.", reason="The reason for banning the user.")
+    @app_commands.describe(user="The user you want to ban from scrims.", reason="The reason for unbanning the user.")
```

Comment:
~~The user you want to ban from scrims.~~

The user you want to ban from scrims.  ✅

Reply:
sorry miss that part, it looks god now, right [deadaf](https://github.com/deadaf) ?